app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreatePartialShipmentGroupedProductCustomStock.xml

```diff
@@ -8,7 +8,7 @@
 
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
-    <test name="AdminCreatePartialShipmentGroupedProductCustomStock" extends="AdminCreateCreditMemoTotalRefundGroupedProductDefaultStock">
+    <test name="AdminCreatePartialShipmentGroupedProductCustomStock" extends="additionalStockAdminCreateCreditMemoTotalRefundFullInvoiceGroupedProductWithDefaultStock">
```

Comment:
Is name capitalization part of convention?

Reply:
Will be fixed in next PR

Reply:
Will be fixed in next PR